Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactions+Types.swift

```diff
@@ -33,3 +67,86 @@ extension ChatMessageReactionsBubbleStyle {
         self == .bigIncoming || self == .smallIncoming
     }
 }
+
+/// Default reactions sorting provided by Stream.
+public enum ReactionSorting {
+    /// Sorting by score.
+    public static func byScore() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
+        { lhs, rhs in
+            lhs.score > rhs.score
+        }
+    }
+
+    /// Sorting by count.
+    public static func byCount() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
+        { lhs, rhs in
+            lhs.count > rhs.count
+        }
+    }
+
+    /// Sorting by firstReactionAt.
+    public static func byFirstReactionAt() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
+        { lhs, rhs in
+            guard let lhsFirstReactionAt = lhs.firstReactionAt, let rhsFirstReactionAt = rhs.firstReactionAt else {
+                return false
+            }
+
+            return lhsFirstReactionAt < rhsFirstReactionAt
+        }
+    }
+
+    /// Sorting by lastReactionAt.
+    public static func byLastReactionAt() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
+        { lhs, rhs in
+            guard let lhsLastReactionAt = lhs.lastReactionAt, let rhsLastReactionAt = rhs.lastReactionAt else {
+                return false
+            }
+
+            return lhsLastReactionAt > rhsLastReactionAt
+        }
+    }
+
+    /// Sorting by firstReactionAt and count.
+    public static func byFirstReactionAtAndCount() -> (ChatMessageReactionData, ChatMessageReactionData) -> Bool {
```

Comment:
The downside is that you have to manually provide all combinations for sorting. In video, we have a more generic approach: https://github.com/GetStream/stream-video-swift/blob/main/Sources/StreamVideo/Utils/Sorting.swift. You can combine different comparators as needed.